packages/react-charting/docs/TestPlans/LineChart/ComponentTests.md

```diff
@@ -0,0 +1,67 @@
+**Line Chart – Component test plan**
+
+**Sub-components: Line, x-axis, y-axis, Event, Time Range, Callout, Drop down and Legend**
+
+1. **Line: Line data, Line color (multi colors), Line label (show/hide)**
+1. **Legends: show/hide legends, highlight the corresponding line on legend hover**
+1. **Callout: Default/custom callout**
+1. **Labels: x-Axis labels**
+
+|**Test steps**|**Validation**|**Tool used**|
+| :-: | :-: | :-: |
+|Test 1: [Snapshot testing]|||
+|- With only data prop, numerical data on x-axis.|Renders line chart correctly|RTL|
+|- With only data prop, date data on x-axis.|Renders line chart correctly|RTL|
+|- With HideLegend prop set to “true”|Should hide legends|RTL & Enzyme|
+|- With HideTooltip prop set to “true”|Should hide the tooltip in chart|RTL & Enzyme|
+|- With EnabledLegendsWrapLines set to “true”|Should enable the legends to wrap lines if there is not enough space to show all legends on a single line|Enzyme|
+|- With ShowXAxisLablesTooltip set to “true”|Should truncate x axis labels and show tooltip on x axis labels|Enzyme|
+|- With WrapXAxisLables set to “true”|Should wrap x axis label values|Enzyme|
+|- With yAxisTickFormat set to “%d”|<p>Should render the y-axis ticks in the format specified</p><p></p>|Enzyme|
+|Test 2: Basic props testing|||
+|- HideLegend prop set to “true”|Should not mount legend when hideLegend is true|Enzyme|
+|- HideLegend prop set to “false”|Should mount legend when hideLegend is false|Enzyme|
+|- HideTooltip prop set to “true”|Should not mount callout when hideTootip is true|Enzyme|
+|- HideTooltip prop set to “false”|Should mount callout when hideTootip is false|Enzyme|
+|- onRenderCalloutPerStack prop is not given|Should not render onRenderCalloutPerStack|Pending|
+|- onRenderCalloutPerDataPoint is given|Should render onRenderCalloutPerDataPoint|Pending|
+|- onRenderCalloutPerDataPoint is given|Should not render onRenderCalloutPerDataPoint|Pending|
```

Comment:
onRenderCalloutPerDataPoint is not given

Reply:
updated